deployer/__main__.py

```diff
@@ -97,6 +114,7 @@ def main():
     build_parser = subparsers.add_parser("build")
     deploy_parser = subparsers.add_parser("deploy")
     validate_parser = subparsers.add_parser("validate")
+    deploy_support_parser = subparsers.add_parser("deploy-support")
```

Comment:
Are you envisioning some cases where we do not deploy-support by default?

Reply:
Ping 😉

Reply:
@damianavila I think for small clusters like farallon, maybe. Although experience with carbonplan suggests otherwise.

I made it a separate command primarily because `deploy` can be used to deploy individual hubs as well, not just an entire cluster. I've added this as a CD step now, and we should be able to deploy this to all hubs that are continuously deployed.